test/jdk/sun/java2d/SunGraphics2D/DrawImageBilinear.java

```diff
@@ -156,63 +144,58 @@ private static void testRegion(BufferedImage bi,
         }
     }
 
-    private static boolean skipOglTextureTest = false;
-
-    public static void main(String[] args) {
-        boolean show = false;
-        for (String arg : args) {
-            if ("-show".equals(arg)) {
-                show = true;
-            }
-        }
-
-        String arch = System.getProperty("os.arch");
-        boolean isOglEnabled = Boolean.getBoolean("sun.java2d.opengl");
-        skipOglTextureTest = false;
-        System.out.println("Skip OpenGL texture test: " + skipOglTextureTest);
-
-        DrawImageBilinear test = new DrawImageBilinear();
-        Frame frame = new Frame();
+    private static void createAndShowGUI() {
+        test = new DrawImageBilinear();
+        frame = new Frame();
         frame.add(test);
+        frame.setBackground(Color.WHITE);
```

Comment:
How the specific background color helps the test to pass, as far as I understand it should not affect the test, no?

Reply:
I noticed in my local Ubuntu VM that sometimes the background was not pure white and had some delta. Thats why i explicitly added background color. But looks like it is not needed and test passes all the time in CI, so i have removed it.